src/clims/migrations/0001_initial.py

```diff
@@ -172,7 +172,6 @@ class Migration(migrations.Migration):
             options={
                 'db_table': 'clims_organizationfile',
             },
-            bases=(clims.models.file.StructuredFileMixin, models.Model),
```

Comment:
Just a note that in the future we shouldn't change old migrations, since it means we can run them over existing data. At this stage we can just reset the db, though, so it shouldn't be a problem.

Reply:
make reset-db actually failed when the class StructuredFileMixin was removed. If it were existing data in db, it seems that we were to keep an empty class named StrucutedFileMixin in order to the migration to succed?

Reply:
I'm not sure I understand. Did you create a new migration script?